### PR TITLE
Change stream reset code to 8 bits.

### PR DESCRIPTION
**Pull request by vasilvv**
Addresses #235.

***
<a href="https://pr-preview.s3.amazonaws.com/vasilvv/web-transport/pull/238.html" title="Last updated on Mar 31, 2021, 5:01 PM UTC (b94fd68)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/238/dc59f89...vasilvv:b94fd68.html" title="Last updated on Mar 31, 2021, 5:01 PM UTC (b94fd68)">Diff</a>

### CODE REVIEW

**Comment by vasilvv** [comment]
cc @yutakahirano @ricea

**Review by jan-ivar — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -844,7 +844,7 @@ in either a RST_STREAM frame or a STOP_SENDING frame).
 
 <pre class="idl">
 dictionary StreamAbortInfo {
-  unsigned long long errorCode = 0;
+  octet errorCode = 0;
```
Comment (ricea) [inline index.bs:847]: I think we should probably use [EnforceRange] here. The default modulo behaviour would be confusing, and [Clamp] would probably be unhelpful.

**Review by ricea — COMMENTED** [review]
Unless I'm misreading this, the title should be "Change stream reset code to 8 bits".

**Review by ricea — APPROVED** [review]
lgtm

**Pull request merged**